gnosis/cowswap
# add cases of the tooltip visibility

# Summary

Fixes #1816 

total trades tooltip is always visible, and it shouldnt be visible when:
-a user is not connected
-a user is not connected to the Mainnet
-totally new user without trades

  # To Test

1. Open the page `Profile`
2. follow the conditions below to see the tooltip works when user is connected, the network is mainnet, trades numbers is higher than 0
